### ISSUE TITLE
Cucmber HTML Report is having ANSI values

### ISSUE THREAD

**Issue by varshanharshank** [WasiqB/multiple-cucumber-html-reporter#314]
**Environment (please complete the following information):**
 - multiple-cucumber-html-reporter: 3.6.0
 - Node.js version: v18.17.1
 - NPM version: 9.6.7
 - Platform name and version: macOs sonoma 14.3
 - Cucumber version: [e.g. 4.2.0]

**Config of multiple-cucumber-html-reporter**
I configured the report using the below file

//cucumberReport.ts

import report = require('multiple-cucumber-html-reporter')
report.generate({
  jsonDir: 'reports/cucumberReport',
  reportName: 'Purple Portal Report',
  reportPath: './reports/cucumberReport',
  metadata: {
    browser: {
      name: '',
      version: '',
    },
    device: 'Local test machine',
    platform: {
      name: '',
      version: '',
    },
  },
  customData: {
    title: 'Run info',
    data: [
      { label: 'Project', value: 'CRSP' },
      { label: 'Application', value: 'Purple Portal' },
    ],
  },
})

**Describe the bug**
In the Html report, assertion error is showing with ANSI Values. But, terminal it is displaying correctlry
**Terminal Output**
<img width="656" alt="image" src="https://github.com/WasiqB/multiple-cucumber-html-reporter/assets/81335781/feaf3c85-178c-4fdd-af58-7334da93b076">

**Cucumber Html Report**

<img width="1182" alt="image" src="https://github.com/WasiqB/multiple-cucumber-html-reporter/assets/81335781/7c84837e-0353-4170-9688-03494fe84f64">

**To Reproduce**
1. Project is configured with Playwright
2. running the command with the following command **npx cucumber-js test --tags '@smoke'**
3. cucumber.json files is given below
//cucumber.json
{
  "default": {
    "paths": ["src/test/features"],
    "require": ["src/test/stepDefinitions/*.ts", "src/hooks/hooks.ts"],
    "requireModule": ["ts-node/register"],
    "publishQuiet": true,
    "format": [
      "progress-bar",
      "./src/helper/report/allureReport.ts:./output.txt",
      "html:reports/cucumberReport/cucumber-report.html",
      "json:reports/cucumberReport/cucumber-report.json"
    ],
    "parallel": 1
  }
}
**package.json**
<img width="515" alt="image" src="https://github.com/WasiqB/multiple-cucumber-html-reporter/assets/81335781/848d7953-0f66-4936-998d-090ea9ff6a23">

[Include code or an example repository that can easily be set up]

**Expected behavior**
ANSI Values should not be present in the html file

**Log**
If applicable, add logs to help explain your problem. If you don't have logs, enable 'debug:true' in the config and paste the log here.
Please use proper markdown to style it

**Additional context**

**Comment by varshanharshank** [comment]
@WasiqB any update about this issue

**Comment by WasiqB** [comment]
Hi @varshanharshank , did you test this issue using the latest version 3.6.2?

Let me know if the issue still exists.

**Comment by varshanharshank** [comment]
@WasiqB Yes itried with 3.6.2 version and still seeing this issue. Refer the below info

In terminal, it is showing the text properly but in the repot it is not showing properly

CLI screen
<img width="890" alt="image" src="https://github.com/WasiqB/multiple-cucumber-html-reporter/assets/81335781/ec40171d-5d56-4b7a-a845-391e437da798">


Report

<img width="1355" alt="image" src="https://github.com/WasiqB/multiple-cucumber-html-reporter/assets/81335781/5c4b5999-9b5f-49d1-b5ab-2408dd8796a4">

**Comment by WasiqB** [comment]
Hi @varshanharshank ,

Check this issue with the new version 4.0.0-beta.0 and update the ticket if this issue is still relevant.
